editoast/src/views/train_schedule/mod.rs

```diff
@@ -0,0 +1,833 @@
+use std::collections::HashSet;
+
+use crate::core::{AsCoreRequest, CoreClient};
+use crate::error::{InternalError, Result};
+use crate::models::{
+    Create, Curve, Delete, Pathfinding, PathfindingPayload, ResultPosition, ResultSpeed,
+    ResultStops, Retrieve, RollingStockModel, RoutePath, Scenario, SignalSighting,
+    SimulationOutput, SimulationOutputChangeset, Slope, TrainScheduleChangeset, Update, ZoneUpdate,
+};
+use crate::models::{ResultTrain, Timetable, TrainSchedule};
+use crate::schema::rolling_stock::RollingStock;
+use crate::schema::utils::Identifier;
+use crate::DieselJson;
+
+use crate::DbPool;
+use actix_web::dev::HttpServiceFactory;
+use actix_web::web::{self, Data, Json, Path, Query};
+use actix_web::{delete, get, patch, post, HttpResponse};
+use diesel::Connection;
+use editoast_derive::EditoastError;
+use geos::geojson::JsonValue;
+use itertools::izip;
+use reqwest;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+use super::electrical_profiles::ElectricalProfileSet;
+
+mod projection;
+use self::projection::Projection;
+
+#[derive(Serialize, Debug, PartialEq)]
+pub struct TrainScheduleDetails {
```

Comment:
Where is that used and why duplicate `models::train_schedule::TrainScheduleDetails`?

Reply:
Ack. To be fair it was there before ;)